agent/data/table.hpp

```diff
@@ -90,7 +90,7 @@ template <typename ItemType> class Table
         // Drop sensors if it not present in answer
         for (auto it = _items.begin(); it != _items.end();)
         {
-            if (data.find((*it)->path) == data.end())
+            if (data.find(_path + "/" + (*it)->name) == data.end())
```

Comment:
Construct path (_path + "/") part outside the cycle.

Reply:
done